docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.7.2-alpine
+FROM ruby:3.0.3-alpine3.15
```

Comment:
Is there a reason we are fixing the alpine version? Doing so restricts us from getting patches

Reply:
Thanks, will change. I didn't realise that was the reason.